src/test/crimson/gtest_seastar.cc

```diff
@@ -15,25 +15,35 @@ SeastarRunner seastar_test_suite_t::seastar_env;
 
 int main(int argc, char **argv)
 {
-  ::testing::InitGoogleTest(&argc, argv);
+  // preprocess args
+  std::vector<const char*> args;
+  bool global_log_level_is_set = false;
+  const char* prefix_log_level = "--default-log-level";
+  for (int i = 0; i < argc; ++i) {
+    if (std::strncmp(*argv, prefix_log_level,
+                     std::strlen(prefix_log_level)) == 0) {
+      global_log_level_is_set = true;
+    }
+    args.push_back(*argv);
+    ++argv;
+  }
+  // HACK: differentiate between the `make check` bot and human user
+  // for the sake of log flooding
+  if (!global_log_level_is_set && !std::getenv("FOR_MAKE_CHECK")) {
+    std::cout << "WARNING: set default seastar log level to debug" << std::endl;
+    ++argc;
+    args.push_back("--default-log-level=debug");
+  }
+
+  auto app_argv = const_cast<char**>(args.data());
+  ::testing::InitGoogleTest(&argc, app_argv);
```

Comment:
```suggestion
  ::testing::InitGoogleTest(&argc, argv);
```

Reply:
consolidated the users to consume the preprocessed app_argc and app_argv